.gitlab-ci.yml

```diff
@@ -13,9 +13,10 @@ stages:
   - configure
   - compile
   - unit_test
-  - dev_test
   # - juelich_tests
```

Comment:
@chraibi is the .gitlab-ci still needed or should we remove it as we are no longer working on gitlab?

Reply:
@schroedtert I would remove it.